boards/arm/nrf52/nrf52-feather/Kconfig

```diff
@@ -5,4 +5,54 @@
 
 if ARCH_BOARD_NRF52_FEATHER
 
-endif
+menuconfig NRF52_FEATHER_KB_FEATHERWING
+	bool "Keyboard FeatherWing"
+    default n
+    ---help---
+        Enable support for the SolderParty FeatherWing
+        https://www.solder.party/docs/keyboard-featherwing/
+
+if NRF52_FEATHER_KB_FEATHERWING
+config KB_FEATHERWING_LCD
```

Comment:
@patacongo what are your thoughts here on how this calls select? Originally I had the LCD dependent on LCD_ILI9341 and no selects, but here it seems like you would want as a user to enable the top level hardware you have and have it turn on the required hardware.

I am fine changing this back if you think that makes sense.  I just was thinking this would make is easy for a user.

Reply:
> 
> 
> @patacongo what are your thoughts here on how this calls select? Originally I had the LCD dependent on LCD_ILI9341 and no selects, but here it seems like you would want as a user to enable the top level hardware you have and have it turn on the required hardware.
> 
> I am fine changing this back if you think that makes sense. I just was thinking this would make is easy for a user.

I agree with you.  FEATHERWING is like a "macro" that just automatically supports everything you need for the board.

One question:  The use of the FeatherWing does not depend on NRF really.  It could be selected for any Feather board potentially.  So I wonder if this does not below in drivers/lcd and if it should not be named LCD_FEATHER_KB_FEATHERWING.

Reply:
> One question:  The use of the FeatherWing does not depend on NRF really.  It could be selected for any Feather board potentially.  So I wonder if this does not below in drivers/lcd and if it should not be named LCD_FEATHER_KB_FEATHERWING.
> 

If we do that what would the difference be between LCD_FEATHER_KB_FEATHERWING and LCD_ILI9341?  Seems like this would belong in `boards/` maybe in an expansion boards folder where there would be Kconfigs for different addon hardware for devkits?

Reply:
> 
> 
> > One question:  The use of the FeatherWing does not depend on NRF really.  It could be selected for any Feather board potentially.  So I wonder if this does not below in drivers/lcd and if it should not be named LCD_FEATHER_KB_FEATHERWING.
> 
> If we do that what would the difference be between LCD_FEATHER_KB_FEATHERWING and LCD_ILI9341? Seems like this would belong in `boards/` maybe in an expansion boards folder where there would be Kconfigs for different addon hardware for devkits?

Yeah, drivers/lcd is not the perfect place.  But the point is that the featherwing is usable across all feather architectures.